### ISSUE TITLE
Meaning of variable "presHCF"

### ISSUE THREAD

**Issue by s-abhyankar** [openhie/covid-ig#6]
I can't tell what this variable means based on its name. If it means hospital admission date, then the mapping is correct to LOINC 8656-1 (Hospital admission date). Otherwise need more info.

http://build.fhir.org/ig/openhie/covid-ig/branches/master/StructureDefinition-WhoCrSurveillanceDataDictionary-mappings.html

Update: I see now in the definitions (http://build.fhir.org/ig/openhie/covid-ig/branches/master/StructureDefinition-WhoCrSurveillanceDataDictionary-definitions.html) that it does mean hospital admission date, but the variable name is not very obvious.

**Comment by ElliotSilver** [comment]
I agree the name is not clear. I took it from the WHO spreadsheet. I assume it means "presented at a healthcare facility." Originally, the variable names matched the spreadsheet exactly, but we ended up deviating from that a bit, and have since decided to include a mapping to the spreadsheet names. 

I'll update the model to use "admitDate" instead of "presHCF" for two elements, one in clinical status/patient course, the other in outcome/patient course.
